client/SDL/SDL3/sdl_clip.cpp

```diff
@@ -242,7 +242,7 @@ UINT sdlClip::MonitorReady(CliprdrClientContext* context, const CLIPRDR_MONITOR_
 		return ret;
 
 	clipboard->_sync = true;
-	SDL_ClipboardEvent ev = { SDL_EVENT_CLIPBOARD_UPDATE, 0, 0, false, NULL, 0 };
+	SDL_ClipboardEvent ev = { SDL_EVENT_CLIPBOARD_UPDATE, 0, 0, false, 0, 0 };
```

Comment:
AFAICT this seems wrong, this field is a pointer not an integer...

Reply:
ok, the order is wrong, correct.
```
typedef struct SDL_ClipboardEvent
 {
     SDL_EventType type; /**< SDL_EVENT_CLIPBOARD_UPDATE */
     Uint32 reserved;
     Uint64 timestamp;   /**< In nanoseconds, populated using SDL_GetTicksNS() */
     bool owner;       /**< are we owning the clipboard (internal update) */
     Sint32 n_mime_types;     /**< number of mime types */
     const char **mime_types; /**< current mime types */
 } SDL_ClipboardEvent;
 ```